sdk/src/main/java/co/omise/android/ThreeDSConfigProvider.kt

```diff
@@ -0,0 +1,33 @@
+import android.net.Uri
+import co.omise.android.AuthorizingPaymentURLVerifier
+import co.omise.android.api.Client
+import com.netcetera.threeds.sdk.api.exceptions.InvalidInputException
+import java.net.URL
+
+internal class ThreeDSConfigProvider(private val urlVerifier: AuthorizingPaymentURLVerifier, private val client: Client) {
+    // Use the authorize_uri to create the config url
+    fun createThreeDSConfigUrl(authUrl: String): String {
+        try {
+            // Check if the authUrl is valid
+            URL(authUrl)
```

Comment:
Can we maybe change `authUrl` to `authorizeUri` as per Omise API? Because `authUrl` could be interpreted differently, for example, as Authentication Url or Authorization Url since both of them start with `Auth`.
<img width="133" alt="image" src="https://github.com/omise/omise-android/assets/122003737/ca66a911-9ff4-4af9-9d49-3820e543cd3a">